crates/swc/tests/fixture/issues-5xxx/5112/output/index.js

```diff
@@ -1,7 +1,6 @@
 function c(r, n) {
-    let e = new Uint8Array(4 * r * r);
-    return {
-        e,
+    return new Uint8Array(4 * r * r), {
```

Comment:
This seems wrong

Reply:
This is wrong. It's caused by `unused: false`. However after fix, it's still wrong -- under `unused: false` var declaration should not be drop, but that's another (low proority) issue.